accounts/org.webosports.messaging.gadu/org.webosports.messaging.gadu.json

```diff
@@ -1,16 +1,15 @@
 	{
 		"templateId": "org.webosports.messaging.gadu",
 		"loc_name": "Gadu Gadu",
-		"loc_subheader": "Account Settings",
```

Comment:
You're right, it was not consistent. I think in the end we might need to drop it altogether, but undecided for now. Just want to get some initial work pushed for now so we can take it forward.